synapse/tests/test_cmds_cortex.py

```diff
@@ -127,8 +126,8 @@ class CmdCoreTest(s_t_utils.SynTest):
 
     async def test_log(self):
         async with self.getTestDmon('dmoncore') as dmon:
-            dirn = s_scope.get('dirn')
-            with self.setSynDir(dirn):
+            with self.getTestDir() as dirn:
+
```

Comment:
This actually called `setSynDir` because the default log behavior populates data there.  We can't drop that otherwise test running will populate data outside of the a test directory.